teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1920 @@
+{
```

Comment:
<p>Proteochemometrics (PCM) consists in the modelling via supervised ML algorithms of a biological endpoint  (e.g. compound activity) ... &gt;</p><p>Proteochemometrics (PCM) models a biological endpoint (e.g. compound activity) via supervised ML algorithms ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>